src/coreclr/vm/comutilnative.cpp

```diff
@@ -2036,6 +2036,13 @@ FCIMPL1(UINT32, MethodTableNative::GetNumInstanceFieldBytes, MethodTable* mt)
 }
 FCIMPLEND
 
+FCIMPL2(FC_BOOL_RET, MethodTableNative::IsEquivalentTo, MethodTable* mta, MethodTable* mtb)
+{
+    FCALL_CONTRACT;
+    FC_RETURN_BOOL(mta->IsEquivalentTo(mtb));
```

Comment:
The contract of `IsEquivalentTo_Worker` is GC_TRIGGER, so HCall instead of FCall should be used here?

Reply:
How to pass `MethodTable` in QCall then? Since it's a pointer into unmanaged runtime, it should be safe to directly pass the pointer?